src/unity/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py

```diff
@@ -132,7 +142,7 @@ def create(input_dataset, target, feature=None,
     dataset = _extensions._drawing_classifier_prepare_data(
         input_dataset, feature) if is_stroke_input else input_dataset
 
-    column_names = ['Iteration', 'Loss', 'Elapsed Time']
+    column_names = ['Training Step', 'Loss', 'Train Accuracy', 'Validation Accuracy', 'Elapsed Time']
```

Comment:
So what happens if the user calls `create` with `validation_set=None`? Is that column just empty? It would probably be better for that column to just not appear.

Reply:
Fixed.